### PR TITLE
#124 - Project Details Block Pattern

### PR DESCRIPTION
**Pull request by khleomix**
**Description**
- #124 Create Project Details Block Pattern

**Screenshots**
**Desktop**
image

**Mobile**
image

**Testing Instructions**
1. Create/Edit a page.
2. Add a pattern -> text -> Project Details
3. Edit where necessary

**Contributors**
khleomix

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
The markup for this pattern was already provided by Bea but since it wasn't on a template I didn't include it in the original PR. Can you please check your markup against hers [here](https://github.com/WordPress/twentytwentyfour/issues/188) to check that it all aligns?

**Comment by khleomix** [comment]
@MaggieCabrera I've updated the pattern to the provided markup. Let me know if that works.

**Review by maneshtimilsina — CHANGES_REQUESTED** [review]
Here are a few of my quick findings. 

- Please use the spacing presets instead of arbitrary values for paddings where needed.
- Use `esc_html_x` with context instead of `esc_html`. You can check `404.php` or other approved patterns for the reference.

File: patterns/project-details.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Title: Project Details
+ * Slug: twentytwentyfour/project-details
+ * Categories: text
+ */
+
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"8vh","bottom":"8vh","right":"6vw","left":"6vw"}}},"backgroundColor":"base","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull has-base-background-color has-background" style="padding-top:8vh;padding-right:6vw;padding-bottom:8vh;padding-left:6vw"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"2em","left":"2em"}}}} -->
```
Comment (MaggieCabrera) [inline patterns/project-details.php]: can we use presets instead here for the padding?

File: patterns/project-details.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"15%"} -->
+<div class="wp-block-column" style="flex-basis:15%"><!-- wp:paragraph {"style":{"typography":{"fontSize":"0.8rem","lineHeight":"1.4"}}} -->
```
Comment (MaggieCabrera) [inline patterns/project-details.php:12]: same about font size presets here

**Review by MaggieCabrera — APPROVED** [review]
I changed the spacing values, let's bring this in and iterate

**Pull request merged**